ch_backup/clickhouse/disks.py

```diff
@@ -59,16 +59,15 @@ def __init__(
                 "Backup contains cloud storage data, cloud-storage-source-bucket must be set."
             )
 
-        self._disks: Dict[str, Disk] = {}
+        self._disks: Dict[str, Dict] = {}
         self._created_disks: Dict[str, Disk] = {}
 
     def __enter__(self):
-        self._disks = self._ch_ctl.get_disks()
+        self._disks = self._ch_config.config["storage_configuration"]["disks"]
```

Comment:
Why do you get rid of using `get_disks()` ?

Reply:
Actually I just move it `copy_part` method. It only one place where we really need it.  To generate conf files in runtime we need to keep configuration of disks in this class. So `_disks` by now contains name of the disk and its configuration to provide ability to create configuration for `clickHouse-server`(ch_backup/clickhouse/disks.py:125) and  `clickhouse-disks`(ch_backup/clickhouse/disks.py:193).